tas/views.py

```diff
@@ -62,6 +69,7 @@ def profile_edit(request):
         "form": form,
         "user": user_info,
         "piEligibility": user_info["piEligibility"],
+        "is_student": "student" in user_info.get("title", "").lower(),
```

Comment:
I think this is a good enough heuristic, we may want to update the USER_TITLES to somehow include which ones should have PI eligibility request disabled if it gets any more complicated. 👍🏻